uber/templates/dept_checklist/hotel_requests.html

```diff
@@ -2,17 +2,6 @@
 {% block title %}Hotel Requests{% endblock %}}
 {% block content %}
 
-<script type="text/javascript">
```

Comment:
Instead of deleting this and the block below, can you set it to be inside an if jinja2 tag? We can add a config variable to configspec.ini called, like, `hotel_requests_url` and then hide these blocks inside `{% if not c.HOTEL_REQUESTS_URL %} {% endif %}` tags. This will also let you use `{{ c.HOTEL_REQUESTS_URL }}` as part of the links, so if we ever decide to update the URL we can do it easier.